src/commands/build/build.ts

```diff
@@ -80,10 +76,12 @@ async function createFinalSasFile(target: Target, streamConfig: StreamConfig) {
   finalSasFileContent += `\n${buildTerm}`
 
   if (streamWeb) {
-    finalSasFileContent += getLaunchPageCode(
-      target.serverType,
-      streamConfig.streamServiceName
-    )
+    if (target.serverType !== ServerType.Sasjs) {
```

Comment:
I think that it would be safer for future development to be explicit about supported `serverType` and instead of `if (target.serverType !== ServerType.Sasjs)` use `if (target.serverType === ServerType.Sas9 || target.serverType === ServerType.SasViya)`. (not critical)

Reply:
agreed - we should be specific on serverTypes, we've had cases where if/else logic has been more complex than it should have been due to this notation

Reply:
updated.